✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
❌  2021-03-09T12:13:43   error: ❌ DB connection err:  Invalid connection string 
 MongoParseError: Invalid connection string
    at parseConnectionString (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongodb/lib/core/uri_parser.js:572:21)
    at connect (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongodb/lib/operations/connect.js:284:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongodb/lib/mongo_client.js:225:5
    at maybePromise (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongodb/lib/utils.js:681:3)
    at MongoClient.connect (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongodb/lib/mongo_client.js:221:10)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/connection.js:833:12
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/connection.js:830:19)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/index.js:345:10
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:5
    at new Promise (<anonymous>)
    at promiseOrCallback (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/helpers/promiseOrCallback.js:30:10)
    at Mongoose._promiseOrCallback (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/index.js:1135:10)
    at Mongoose.connect (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/mongoose/lib/index.js:344:20)
    at Object.dbConnection [as default] (/home/orhanors/Desktop/ME/Capstone/backend/src/startup/db.ts:6:4)
    at Object.<anonymous> (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:12:13)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:21:29   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:16:9
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌  2021-03-09T12:21:33   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:16:9
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:21:56   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌  2021-03-09T12:21:59   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌  2021-03-09T12:22:01   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:23:01   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:25:27   error: Internal Server Error 
 Error: Internal Server Error
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:27:31   error: User 
 undefined
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:27:57   error: User 
 Error: User
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:28:48   error: User Not Found 
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
❌  2021-03-09T12:29:32   error: User Not Found 
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅     info: 🚀 Server is running LOCALLY on PORT:  
 
✅     info: 🚀 Connected to DB... 
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found 2021-03-09T12:32:18   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found 🕗2021-03-09T12:33:38   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:34:01   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:15:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:35:33   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌ error: User Not Found | 🕗 2021-03-09T12:36:00   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:37:42   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌ error: User Not Found | 🕗 2021-03-09T12:37:43   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌ error: User Not Found | 🕗 2021-03-09T12:37:52   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:40:48   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
❌ error: User Not Found | 🕗 2021-03-09T12:40:50   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:45:40   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:317:13)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:275:10)
    at query (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/middleware/query.js:45:5)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
❌ error: User Not Found | 🕗 2021-03-09T12:52:44   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at step (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:33:23)
    at Object.next (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:53)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:4:12)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:13:33
    at newFn (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express-async-errors/index.js:16:20)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at newFn (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express-async-errors/index.js:16:20)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
✅ info: 🚀 Connected to DB...    
 
❌ error: User Not Found | 🕗 2021-03-09T12:52:47   
 Error: User Not Found
    at new ApiError (/home/orhanors/Desktop/ME/Capstone/backend/src/utils/errors/ApiError.ts:6:3)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:8
    at step (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:33:23)
    at Object.next (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:14:53)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:4:12)
    at /home/orhanors/Desktop/ME/Capstone/backend/src/server.ts:13:33
    at newFn (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express-async-errors/index.js:16:20)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/route.js:112:3)
    at newFn (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express-async-errors/index.js:16:20)
    at Layer.handle [as handle_request] (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/orhanors/Desktop/ME/Capstone/backend/node_modules/express/lib/router/index.js:335:12)
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
✅ info: 🚀 Server is running LOCALLY on PORT:     
 
✅ info: 🚀 Connected to DB...    
 
